WooCommerce/woocommerce_rms_normal/class-gateway.php

```diff
@@ -3,10 +3,10 @@ class WC_Molpay_Gateway extends WC_Payment_Gateway {
   
   // Constructor method
   public function __construct() {
-    global $woocommerce;
+    global $woocommerce, $post;
 
     $this->id                 = 'wcmolpay';
-    $this->icon = plugins_url( 'images/Fiuu_Logo-01.png', __FILE__ );
+    $this->icon = plugins_url( 'images/Fiuu_Logo-03.png', __FILE__ );
```

Comment:
upload the image 03 file

Reply:
![image](https://github.com/user-attachments/assets/707c6175-5697-4f19-8ae6-07840a84140b)

file uploaded